proto/osmosis/gamm/v1beta1/types/pool.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package osmosis.gamm.v1beta1;
+package osmosis.gamm.v1beta1.types;
```

Comment:
Can we undo this? Worried about this breaking things

Reply:
agree here, having one package for gamm.v1beta1 is better than nesting it in different directories. You have to think that the longer the package name the added few characters add a few bytes to storage.